src/xrGame/Level.cpp

```diff
@@ -440,7 +440,10 @@ void CLevel::OnFrame()
     if (!GEnv.isDedicatedServer)
     {
         if (g_mt_config.test(mtMap))
+        {
+            R_ASSERT(m_map_manager);
```

Comment:
We don't need to check for `m_map_manager`, but we should fix it's initialization on Linux 😉